Gulpfile.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable */
```

Comment:
Gulp does not have one built in, but it would be trivial to add a command to fire one up.  Personally, I use [http-server](https://www.npmjs.com/package/http-server) when developing locally.  I could very easily wire up an `npm run start` task that would fire up a server.